main/krb5-conf/APKBUILD

```diff
@@ -13,6 +13,7 @@ arch="noarch"
 license="MIT"
 depends=""
 makedepends=""
+options="!check"
```

Comment:
Why? Upstream does not provide any tests? Please always add a short comment after the `!check` why it’s disabled.